### DISCUSSION TITLE
golang deprecated funcs are not marked strikethrough, why?

### DISCUSSION THREAD

**Discussion by Littledarren** [hrsh7th/nvim-cmp#1110] [Q&A]
I wrote a func like this, which is the recommended way to annotate as deprecated
```golang
// xxx do blablabla
//
// Deprecated: use yyy
func xxx(){}
```
and while completing this func, nvim-cmp does not show a strikethrough on the `xxx`.

However, it do shows a strikethrough when i write comments like this
```golang
// Deprecated use yyy
func xxx(){}
```
![image](https://user-images.githubusercontent.com/20529233/182324929-2f2ac93b-196d-4d82-ad6a-cd2e7670a79a.png)

I'vs searched the google and found this https://github.com/golang/go/issues/40447, which told that the deprecated status is not marked by gopls.
I concluded that this mark comes from the completion plugin and it seems to be not that smart as i expected.

----

Anyway, I wonder how can i strikethrough the depracated symbol. Which part of codes should I change?

**Comment by tkgalk** [comment]
Yep, having the same issue. It seems whatever makes the strikethrough (`gopls`?) only does it if the comment starts with `Deprecated:` not when any of the lines start with `Deprecated:` as it should.
